protocol/avdl/keybase1/account.avdl

```diff
@@ -58,9 +58,9 @@ protocol account {
   GetLockdownResponse getLockdownMode(int sessionID);
   void setLockdownMode(int sessionID, boolean enabled);
 
-  // Call an API handler that will send an e-mail to the user with their
+  // Call an API handler that will send an e-mail or sms to the user with their
   // Keybase username.
-  void recoverUsername(int sessionID, string email);
+  void recoverUsername(int sessionID, string email, PhoneNumber phone);
```

Comment:
it will be either username or phone, right? so maybe we can use a variant here

Reply:
the endpoint allows for one or both right now